app/services/channels/adapters/discord.py

```
@@ -0,0 +1,114 @@
+        self._stopping = True
+        if self._client is not None:
+            try:
+                await self._client.close()
+            except Exception:  # noqa: BLE001
+                logger.exception("error closing discord client")
+        if self._task is not None:
+            self._task.cancel()
+            try:
+                await self._task
+            except (asyncio.CancelledError, Exception):  # noqa: BLE001
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)